docs/examples/input-tag/tag.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <el-radio-group v-model="tagType" aria-label="type control" class="mr-5">
+      <el-radio-button
+        v-for="item in type"
+        :key="item"
+        :label="item"
+        :value="item"
+      />
+    </el-radio-group>
+    <el-radio-group v-model="tagEffect" aria-label="effect control">
+      <el-radio-button
+        v-for="item in effect"
+        :key="item"
+        :label="item"
+        :value="item"
+      />
+    </el-radio-group>
```

Comment:
```suggestion
    <el-segmented v-model="tagType" :options="type" />
    <el-segmented v-model="tagEffect" :options="effect" style="margin-left: 1rem" />
```

Reply:
The same effect `segmented` can use more concise code, which makes the demo easier to use.

![image](https://github.com/user-attachments/assets/cea689f2-bdbb-4c21-ad6e-a0989b4f586a)